src/enstore_plotter_module.py

```diff
@@ -62,8 +108,13 @@ def roundtime(seconds, rounding=None):
     """
     Round the provided time and return it.
 
-    `seconds`: time in seconds.
-    `rounding`: None, 'floor' or 'ceil'.
+    Args:
+        seconds: time in seconds.
+        rounding: None, 'floor' or 'ceil'.
+    Returns:
+        time in seconds with indicated rounding applied .
```

Comment:
So does this function do nothing if no `rounding` argument is supplied?

Reply:
That appears to be the case.  This function only seems to be called with 'ceil' in the code